apps/server/src/modules/board/service/board-do-authorizable.service.ts

```diff
@@ -72,7 +75,7 @@ export class BoardDoAuthorizableService implements AuthorizationLoaderService {
 					userId: user.id,
 					firstName: user.firstName,
 					lastName: user.lastName,
-					roles: [BoardRoles.READER],
+					roles: isDrawingElement ? [BoardRoles.EDITOR] : [BoardRoles.READER],
```

Comment:
What is the default value of this boolean? 
For me is this change unclear, can we speak about it? 
With this line you create a exception for one element that invert the authorisation logic/role), it look like the first step into the wrong direction.

Reply:
The default is false. 

We want to store uploaded assets from tldraw in s3 storage - by default every authorized user is supposed to be able to upload to tldraw. There is already a logic in place to upload files to column board nodes. If requested board node is an instance of DrawingElement (which is Tldraw) we want to give student role access to upload.

The alternative I can think of is to create new authorizable service only for DrawingElements which would basically be a copy paste of board-do-authorizable.service with just this one line changed.